.github/workflows/release.yml

```diff
@@ -20,6 +20,7 @@ jobs:
         registry-url: https://registry.npmjs.org
     - name: Build and Test
       run: |
+        echo "//registry.npmjs.org/:_authToken=${NPM_TOKEN}" > .npmrc
```

Comment:
Last build failed - based on the docs, I thought semantic-release would pick up the token from the environment variable, but I'm adding this to see if it picks it up from there

Reply:
And I did confirm that this repo is configured for the `npm` environment in `repo-standard`, so this token should be available

Reply:
Nice change. If the `setup-node` action sets an `.npmrc` file (seems possible), then it will override our token.

I actually tried to fix the semantic-release docs around this a while back 😂, because they were wrong. Even with my fix, they're still not great :/

https://github.com/semantic-release/npm/pull/413